gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPickerUiState.kt

```diff
@@ -12,7 +12,7 @@ internal data class AvatarPickerUiState(
     val isLoading: Boolean = false,
     val error: SectionError? = null,
     val profile: ComponentState<Profile>? = null,
-    val identityAvatars: IdentityAvatars? = null,
+    val identityAvatars: EmailAvatars? = null,
```

Comment:
⛏ It's an internal class, but I think it's better to eliminate all `identity` naming.

Reply:
Yes, good catch.